src/components/HeaderNavMenu.jsx

```diff
@@ -53,26 +47,22 @@ function HeaderNavMenu() {
         </Grid>
       ))}
       <Grid item className={classes.avatarMenu}>
```

Comment:
Forgot to update this to use the `authItem` CSS class

Reply:
Still need this

Reply:
ok change is made